.circleci/config.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       SERVER_ID: "0"
       PLAYWRIGHT_BROWSERS_PATH: "0" # install playwright browsers to node_modules for caching
     working_directory: ~/action
-    resource_class: medium+
+    resource_class: large
```

Comment:
We should discuss this change. Right now, the tests take quite a long time to run on CI (~30 minutes in total). This is significantly slower than on my local machine, where tests run in about 5 minutes.

I think most will agree that 30 minutes is far too long to wait on each push. So, a few ideas:

1. We could make the integration test only run on your default branch.
2. We could create a manually triggered workflow to run these tests before deployment.
3. We could restructure or remove some tests that take a long time to run (e.g., waiting for the demo "<user> is typing reactions"). For instance, maybe in test mode we make the demo interactions occur with a shorter delay. 
4. We could reduce the number of browsers we test against, maybe even based on which branch we're building. For instance, maybe we just run against one Desktop browser and one Mobile browser on branches, then run a more comprehensive test suite on the default branch.

Reply:
Alright, I reduced the testing scope and removed Firefox and Desktop Safari. With those changes, it's currently taking ~5 minutes to run the test suite against Chrome and Mobile Safari. Does that sound reasonable?